include/envoy/upstream/outlier_detection.h

```diff
@@ -42,6 +42,9 @@ enum class Result {
  */
 class DetectorHostMonitor {
 public:
+  // Types of Success Rate monitors.
+  using SuccessRateMonitorType = enum { externalOrigin, localOrigin };
```

Comment:
nit: switch to `enum class`, also `ExternalOrigin` and `LocalOrigin`

Reply:
Changed to scoped _enum_. Thanks for catching it.